src/routes/(app)/events/[slug]/+page.svelte

```diff
@@ -24,7 +26,9 @@
 </script>
 
 <svelte:head>
-  <title>{event.title} | D-sektionen</title>
+  <title
+    >{languageTag() === "en" && event.titleEn ? event.titleEn : event.title} | D-sektionen</title
```

Comment:
I think this is done automatically way down in the abstraction layers so to say. Basically, depending on the language, "event.title" either represents the data stored in "title" OR "titleEn". I think it's done in one of the hooks in the root folder.